docker-compose.yml

```diff
@@ -43,6 +51,18 @@ services:
     depends_on:
       - conjur
 
+  client-dap:
+    image: cyberark/conjur-cli:5
+    entrypoint: sleep
+    command: infinity
+    environment:
+      CONJUR_APPLIANCE_URL: https://cuke-master
+      CONJUR_ACCOUNT: cucumber
+      CONJUR_AUTHN_LOGIN: admin
+      CONJUR_SSL_CERTIFICATE:
```

Comment:
Leaving this variable unset without prior configuration of CLI makes it unnecessary here since you have a "catch-22" situation here:}
- You need the cli to get the certificate
- You need the certificate to start this cli container

Reply:
is this true, though? she gets the certificate one time by running `cat conjur.pem` on the conjur master (eg she doesn't need the cli to get the cert, and she uses that cert to configure the CLI)

Reply:
Yeah you're right though it seems that this step isn't necessary if we just do a `conjur init`. Seems like we keep fetching this cert in different ways and in multiple places so it would be good to collapse that to some common code.